components/push/android/src/test/java/mozilla/appservices/push/PushTest.kt

```diff
@@ -58,7 +64,8 @@ class PushTest {
         return PushManager(
                 sender_id = mockSenderId,
                 bridge_type = BridgeTypes.TEST,
-                registration_id = "TestRegistrationId"
+                registration_id = "TestRegistrationId",
+                database_path = dbFolder.toString()
```

Comment:
So, database_path the directory where to place the DB?

Reply:
Actually, it's the name of the database file. This is a bit more clear in the latest patch.

Sorry for the delay, was fighting with a few issues around the build and android studio.